cache/src/convertable.rs

```diff
@@ -22,41 +21,50 @@ pub trait Convertable {
 	/// Generally, it fails if the accessible item does not implement to accessible interface.
 	/// This shouldn't be possible, but this function may fail for other reasons.
 	/// For example, to convert a [`zbus::Proxy`] into a [`Self::Accessible`], it may fail to create the new [`atspi_proxies::accessible::AccessibleProxy`].
-	async fn to_accessible(&self) -> Result<AccessibleProxy, Self::Error>;
+	fn to_accessible(
+		&self,
+	) -> impl Future<Output = Result<AccessibleProxy, Self::Error>> + Send;
 	/// Creates an [`Self::Action`] from the existing accessible item.
```

Comment:
broken documentation, according to rustdock, just letting you know in case you didn't see it. I skipped past some of them already, but yeah, this is the one I'm noting as an example